lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -890,18 +888,13 @@
       }
 
       &.course-status-processing {
-        background-color: $gray-l5;
-        border: 0;
+
       }
 
       &.course-status-certnotavailable {
-        background-color: $gray-l5;
-        border: 0;
```

Comment:
Have we tested to make sure these cases aren't ever triggered? Just wanted to double check this.